packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -49,7 +49,6 @@ export const getRole = (state: CallClientState, props: CallingBaseSelectorProps)
  * @private
  */
 export const isHideAttendeeNamesEnabled = (state: CallClientState, props: CallingBaseSelectorProps): boolean => {
-  /* @conditional-compile-remove(hide-attendee-name) */
```

Comment:
We don't want to remove these comments until after the feature goes out. we need a quick and easy way to strip the feature if there is something wrong (I'm sure its all happy just a process thing)